docs/initial-milestones.md

```diff
@@ -0,0 +1,65 @@
+# OpenTelemetry: A Roadmap to Convergence
+
+This document covers the intial milestones for each SIG developing an implementation of the OpenTelemetry Specification. For each new language, we want to quickly achieve parity with existing OpenTracing and OpenCensus implementations.
+
+[Merging OpenTracing and OpenCensus: A Roadmap to Convergence](https://medium.com/opentracing/a-roadmap-to-convergence-b074e5815289) defines the goals and timelines for OpenTelemetry.
+
+If you are interested in starting a new SIG, please do the following:
+* [create an issue](github.com/open-telemetry/community/issues) proposing the new SIG.
+* [Configure the new repository](docs/how-to-configure-new-repository.md).
+* Add the milestones listed below to the backlog.
+
+## Initial Milestones
+For languages which have both an OpenTracing and OpenCensus implementation, we would like to achieve parity in OpenTelemetry by **September, 2019**, and sunset the existing OpenTracing and OpenCensus SIGs by **November, 2019**. 
+
+Parity can be defined as the following milestones:
+* A set of interfaces which define the OpenTelemetry Specification in a given programming language.
```

Comment:
how interfaces can define the specification? And why specification word starts with upper case? Do you mean set of interfaces implementing OpenTelemetry spec?

Reply:
I mean, we need to translate the official specification into interfaces (or equivalent language constructs). So these interfaces are the definition the OpenTelemetry specification for this language. Could probably be worded more clearly. :)

Reply:
Cleaned this language up a bit. "A set of interfaces which implement the OpenTelemetry specification in a given programming language."